build/webpack/webpack.services.js

```diff
@@ -3,23 +3,22 @@ var fs = require("fs");
 var glob = require("glob");
 var glob_entries = require('webpack-glob-entries');
 
-//not used?
+
 function getDirectories(srcpath) {
 	return fs.readdirSync(srcpath).filter(function (file) {
 		return fs.statSync(path.join(srcpath, file)).isFile();
 	});
 }
 
 var services = glob_entries(path.join(__dirname, '../../', "/src/services/**/*.js"));
-var clients = glob_entries(path.join(__dirname, '../../', "/src/clients/**/*.js"));
+
```

Comment:
Did you mean to remove your client work?

Reply:
yep - we're not going to support it or user implemented clients so it should go.